lib/src/metta/runner/stdlib.metta

```diff
@@ -45,8 +46,14 @@
   (@return "Quoted atom"))
 (: quote (-> Atom Atom))
 
-; unify matches two atoms and returns $then if they are matched
-; and $else otherwise.
+(@doc unify
+  (@desc "Matches two atoms and returns $then if they are matched and $else otherwise")
+  (@params (
+    (@param "First atom to unify with")
+    (@param "Second atom to unify with")
+    (@param "Result if two atoms unified successfully")
+    (@param "Result otherwise")))
+  (@return "Unit atom"))
```

Comment:
1. It returns not "unit atom" but atom of any type.
```suggestion
  (@return "Third argument when first two atoms are matched of forth one otherwise"))
```
2. Below you sometimes used "Unit atom" sometimes used "Unit value" to describe `(->)` type returned. It is better to use same description in all cases.

Reply:
1. Of course. I've copy-pasted this from previous function and forgot to edit result...
2. So I'll replace unit value with unit atom then.